Pipfile.lock

```diff
@@ -157,6 +157,14 @@
             "markers": "python_version >= '3.6'",
             "version": "==39.0.0"
         },
+        "exceptiongroup": {
+            "hashes": [
+                "sha256:327cbda3da756e2de031a3107b81ab7b3770a602c4d16ca618298c526f4bec1e",
+                "sha256:bcb67d800a4497e1b404c2dd44fca47d3b7a5e5433dbab67f96c1a685cdfdf23"
+            ],
+            "markers": "python_version < '3.11'",
```

Comment:
I generated this lockfile on Python 3.10, hence the additional dependencies for < 3.11.